### ISSUE TITLE
Remove `feature.coordinates`

### ISSUE THREAD

**Issue by springmeyer** [mapbox/vt2geojson#5]
We are currently outputting both `feature.coordinates` and `feature.geometry.coordinates`. The former is the result of `layer.feature(i).loadGeometry();` which is a custom structure of x,y in tile coordinates. The latter is geojson coordinates. I think we should output only the latter by removing this line:

```diff
@@ -69,7 +78,6 @@ function readTile(args, buffer, callback) {
         var layer = tile.layers[layerID];
         for (var i = 0; i < layer.length; i++) {
             var feature = layer.feature(i).toGeoJSON(args.x, args.y, args.z);
-            feature.coordinates = layer.feature(i).loadGeometry();
             collection.features.push(feature);
         }
     });
```

/cc @jfirebaugh who might remember if there was a reason for `feature.coordinates = layer.feature(i).loadGeometry();`.

**Comment by jfirebaugh** [comment]
No reason that I'm aware of.

**Comment by mapsam** [comment]
This was resolved in #12 and has been released in `@mapbox/vt2geojson@1.1.3`
